templates/helpers.yaml

```diff
@@ -1,37 +1,41 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
 metadata:
-  name: {{ .Release.Name }}-helper-deployment
+  name: {{ .Release.Name }}-helper-shard
   labels:
-    app: {{ .Release.Name }}-helper
+    app: {{ .Release.Name }}-helper-shard
 spec:
-  replicas: 1
   selector:
     matchLabels:
-      app: {{ .Release.Name }}-helper
+      app: {{ .Release.Name }}-helper-shard
+  serviceName: {{ .Release.Name }}-helper-shard
+  replicas: 1 # number of shards
```

Comment:
I'd rather explain why it is set to 1, because it is quite clear what this parameter means

Reply:
Well in a more general context, in Kubernetes, this is usually the number of exact copies. The load balancer can pick any of them and since they are stateless.

In a statefulset this changes, and replicas stop being the same from each other. They become uniquely addressable. That's why I added the comment.

Reply:
I can remove it, but I wanted to foreshadow where we can tune the number of shards.

Reply:
Can we control the number of shards with this in this future, given that each shard is unique and not an exact copy of each other?

Reply:
Yes. Each will get a special name in the cluster like this:
```
$ kubectl get pods
NAME                READY   STATUS    RESTARTS   AGE
h1-helper-shard-0   1/1     Running   0          2d18h
h1-helper-shard-1   0/1     Pending   0          10s
h2-helper-shard-0   1/1     Running   0          2d18h
h3-helper-shard-0   1/1     Running   0          2d18h
```

They get an ordinal number to identify within the statefulset